app/views/submissions/index.html.erb

```diff
@@ -123,11 +123,12 @@
       </td>
       <td class="exclude-click" style="text-align:center;">
         <% if @autograded and submission.version > 0 then %>
-          <%= link_to "<i class='material-icons'>autorenew</i>".html_safe,
-              [:regrade, @course, @assessment, submission_id: submission.id],
-              { method: :post,
+          <%= button_to [:regrade, @course, @assessment, submission_id: submission.id],
```

Comment:
realized that link_to still does not address the issue (where it uses get instead of post).
Changed to button_to, which is meant for post interactions in the first place.  (in fact it uses post method by default, but I just left the method specification here anyways)